### PR TITLE
Prevent accidental interpretation of `<A>` as HTML

### PR DESCRIPTION
**Pull request by pthariensflame**
Also add code font.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -3,21 +3,21 @@
...
-/// Converts a reference to A into a slice of length 1 (without copying).
+/// Converts a reference to `A` into a slice of length 1 (without copying).
 pub fn ref_slice_mut<A>(s: &mut A) -> &mut [A] {
     unsafe {
         core::slice::from_raw_parts_mut(s, 1)
     }
 }
 
-/// Converts a reference to Option<A> into a slice of length 0 or 1 (without copying).
+/// Converts a reference to `Option\<A\>` into a slice of length 0 or 1 (without copying).
```
Comment (steveklabnik) [inline src/lib.rs]: With the backticks, the \s shouldn't be needed. Could you just include the graves please? Thanks!

**Review by steveklabnik — CHANGES_REQUESTED** [review]

File: src/lib.rs (hunk 1 above)
Comment (pthariensflame) [reply src/lib.rs]: Ok!

**Comment by pthariensflame** [comment]
@steveklabnik How does it look now?

**Review by steveklabnik — APPROVED** [review]

**Comment by steveklabnik** [comment]
Thank you!

**Pull request merged**